tests/testthat/test-template.R

```diff
@@ -778,3 +778,249 @@ test_that("curve_metric_summarizer() deals with characters in truth", {
 
   expect_identical(roc_curve_res, roc_curve_exp)
 })
+
+## dynamic_survival_metric_summarizer -----------------------------------------
+
+# To be removed once brier_survival() is added
+brier_survival_vec <- function(truth,
+                               estimate,
+                               censoring_weights,
+                               eval_times,
+                               na_rm = TRUE,
+                               case_weights = NULL,
+                               ...) {
+  check_dynamic_survival_metric(
+    truth, estimate, censoring_weights, case_weights, eval_times
+  )
+
+  n_distinct_time <- dplyr::n_distinct(eval_times)
+  if (n_distinct_time != 1) {
+    abort(paste0(
+      "`eval_times` should have at most 1 unique value. But ", n_distinct_time,
```

Comment:
```suggestion
      "`eval_times` should have at most 1 unique value, but ", n_distinct_time,
```

Understood that this is a placeholder, but seeing a similar pattern in #347!